File: bench.js
function runCode(number)
{
    if (number === 1)
        return 1;
    else {
        let num1 = number;
        let num2 = number;
        for (let i = 0; i < number * 1000; i++) {
            num2 *= number;
        }
        return num2 * runCode(num1 - 1);
    }
}

runCode(10);

File: bench.py
def run_code(number):
    if number == 1:
        return 1
    else:
        num1 = number
        num2 = number
        for i in range (0, int(number * 1000)):
            num2 *= number
        return num2 * run_code(num1 - 1)

run_code(float(10))
